### PR TITLE
DDCE-1096:Add 'Title' Radio Buttons to Officials and Nominee Journeys

### PR DESCRIPTION
**Pull request by prameya-phatak**
# DDCE-1096

## New feature

Add 'Title' Radio Buttons to Officials and Nominee Journeys

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -754,6 +754,19 @@ bankDetailsSummary.section = (W)Operations and funds(W)
 # ----------------------------------------------------------
 officialsAndNominees.section = (W)Officials and nominees(W)
 
+nameTitle = Title(W)
```
Comment (abhijeetardale) [inline conf/messages.cy:757]: (W) missing as prefix

File: test/controllers/addressLookup/AuthorisedOfficialsAddressLookupControllerSpec.scala [hunk 2]
```diff
@@ -63,7 +63,7 @@ class AuthorisedOfficialsAddressLookupControllerSpec extends SpecBase with Befor
     mockAddressLookupConnector, inject[ErrorHandler], messagesControllerComponents)
 
   private val localUserAnswers: UserAnswers = emptyUserAnswers.set(
-    AuthorisedOfficialsNamePage(0), Name("Jim", Some("John"), "Jones")).success.value
+    AuthorisedOfficialsNamePage(0), Name(SelectTitle.values.head, "Jim", Some("John"), "Jones")).success.value
```
Comment (abhijeetardale) [inline test/controllers/addressLookup/AuthorisedOfficialsAddressLookupControllerSpec.scala:66]: SelectTitle.Mr would be more readable , please check all the places

**Review by abhijeetardale — CHANGES_REQUESTED** [review]

**Review by abhijeetardale — APPROVED** [review]

File: test/controllers/addressLookup/AuthorisedOfficialsAddressLookupControllerSpec.scala (hunk 2 above)
Comment (prameya-phatak) [reply test/controllers/addressLookup/AuthorisedOfficialsAddressLookupControllerSpec.scala]: modified

File: conf/messages.cy (hunk 1 above)
Comment (prameya-phatak) [reply conf/messages.cy]: modified

**Pull request merged**